src/Form/Form.js

```diff
@@ -16,10 +16,14 @@ const Form = props => {
       .replace(/[^\w\s]|_/g, "")
       .replace(/\s+/g, " ");
     const existingItem = dbItems.find(element => {
-      let newElement = element.name
-        .toUpperCase()
-        .replace(/[^\w\s]|_/g, "")
-        .replace(/\s+/g, " ");
+      let newElement;
+      //adding items to the list auto adds one empty item which causes everything to crash when checking for its name without this condition
+      if (element.name !== undefined) {
+        newElement = element.name
+          .toUpperCase()
+          .replace(/[^\w\s]|_/g, "")
```

Comment:
I'd love to see comments about what the regex is doing here.

Reply:
Got it! I'll make sure to add that in the next task.